server/sonar-web/src/main/js/apps/tutorials/onboarding/TokenStep.js

```diff
@@ -161,7 +238,8 @@ export default class TokenStep extends React.PureComponent {
   };
 
   renderResult = () => {
-    const { token, tokenName } = this.state;
+    const { selection, tokenName } = this.state;
+    const token = this.getToken();
 
     if (!token) {
```

Comment:
I'd check for `selection === 'generate'` here. I don't think we ever want to render this in "existing" mode.